documentation/introduction-the-devon-ide.asciidoc

```diff
@@ -0,0 +1,197 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:imagesdir: https://raw.githubusercontent.com/devonfw/getting-started/master/documentation/
+endif::[]
+
+:doctype: book
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
+= Devon IDE
```

Comment:
The whole inital section is confusing to me.
I copied it from the guide, because it's still included in there, but I think we should ask @hohwille if the parts below "Features and Advantages" can be moved to the [ide wiki](https://github.com/devonfw/ide/wiki).
And do we still make a distinction between the "devon distribution" and the "devonfw-ide", or has it all been merged into one product at this point?